experimental/jackPython/astMapping/main.py

```diff
@@ -1,20 +1,69 @@
 import ast
 import astor
 
+"""
+converts python ast operations to common string representation
+TODO: add in boolean logic
+"""
+def pyop_to_str(op):
+    if type(op) == ast.Add:
+        return "+"
+    elif type(op) == ast.Mult:
+        return "*"
+    elif type(op) == ast.Div:
+        return "/"
+    elif type(op) == ast.Sub:
+        return "-"
+
+"""
+converts a python ast BinOp and converts it to a readable string recursively 
+"""
+def binOp_to_str(bop):
+    if type(bop.left) == ast.BinOp:
```

Comment:
Is it not possible for `bop.right` to also be a `BinOp`?

Reply:
Not that I have seen, but I haven't done extensive testing yet.